build.gradle.kts

```diff
@@ -325,34 +339,41 @@ tasks.register<Copy>("runtime")
     mkdir("${project.properties["runtime.local"]}")
     mkdir("${project.properties["runtime.deploy"]}")
 
-    from(layout.buildDirectory.dir("${project.properties["build.dir"]}/${project.properties["final.name"]}.job"))
-    into(layout.buildDirectory.dir("${project.properties["runtime.deploy"]}"))
-
-    from(layout.buildDirectory.dir("${project.properties["runtime.deploy"]}/bin"))
-    into(layout.buildDirectory.dir("src/bin"))
-
-    from(layout.buildDirectory.dir("${project.properties["build.dir"]}/${project.properties["final.name"]}.jar"))
-    into(layout.buildDirectory.dir("${project.properties["runtime.local"]}/lib"))
+    into(layout.projectDirectory)
 
-    from(layout.buildDirectory.dir("${project.properties["runtime.local"]}/lib/native"))
-    into(layout.buildDirectory.dir("lib/native"))
-
-    from(layout.buildDirectory.dir("${project.properties["runtime.local"]}/conf")) {
-        exclude("*.template")
+    into("${project.properties["runtime.deploy"]}") {
+        from(layout.projectDirectory.dir("${project.properties["build.dir"]}/${project.properties["final.name"]}.job"))
+    }
+    into("${project.properties["runtime.deploy"]}/bin") {
+        from(layout.projectDirectory.dir("src/bin"))
+    }
+    into("${project.properties["runtime.local"]}/lib") {
+        // from(
```

Comment:
Why is this copied out? We are trying to replicate the behaviour from the `ant` build. This directory needs to be present and on the classpath in order for Nutch to be able to run.